filters/LiTreeFilter.cpp

```diff
@@ -244,8 +236,18 @@ void LiTreeFilter::segmentTree(PointView& view, PointIdList& Ui,
     Ni.push_back(n0);
 
     // "For iteration i, we classify each point in Ui as Pi or Ni."
+    double tx = view.getFieldAs<double>(Id::X, t0);
+    double ty = view.getFieldAs<double>(Id::Y, t0);
     for (PointId const& ui : Ui)
-        classifyPoint(ui, view, Ni, Pi);
+    {
+        double ux = view.getFieldAs<double>(Id::X, ui);
+        double uy = view.getFieldAs<double>(Id::Y, ui);
+        double d = std::sqrt((ux - tx) * (ux - tx) + (uy - ty) * (uy - ty));
```

Comment:
Not sure why we're taking square roots just for a comparison since we seem to throw the value away after that.

Reply:
Actually meant to do this, both here and above.